.github/workflows/file-format-check.yml

```diff
@@ -0,0 +1,13 @@
+name: Check asset file names
+on: pull_request
+
+jobs:
+  Filenamecheck:
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v2
+      - shell: bash
+        run: |
+          find assets | grep "\.[A-Z]" && echo "Upper case characters found in file endings" && exit 1
```

Comment:
```suggestion
          find assets -type f -printf '%f\n' | grep "\.[A-Z]" && echo "Ensure file extensions are in lowercase." && exit 1
```

Reply:
-type f -printf '%f\n' is not needed here since it only checks everything after the dot